pkg/util/csi/util.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright The Velero Contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"strings"
+
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"github.com/vmware-tanzu/velero/pkg/features"
+)
+
+const (
+	csiPluginNamePrefix = "velero.io/csi-"
+)
+
+func CSIFeatureNotEnabledAndPluginIsFromCSI(backupItemActionName string) bool {
```

Comment:
Since it's already in csi package, is it better to call the func "ShouldSkip"?

Reply:
Thanks. 
The function name is changed to `ShouldSkipAction`.
The parameter name is changed to `actionName`.